public/js/reducers/management.js

```diff
@@ -67,6 +67,7 @@ export default function management(state, action) {
     case actionTypes.CREDIT_CARD_SUBMISSION_ERRORS:
       return Object.assign({}, initialMgmtState, {
```

Comment:
I think there could be more things besides `tab` that we'd want to preserve so it should just inherit from the current state like this:
````
return Object.assign({}, state, {
  cardSubmissionErrors: action.apiErrorResult,
});
````